docs/gettingstarted.rst

```diff
@@ -1,4 +1,50 @@
 Getting started
 ===============
 
-FIXME
+To be able to deploy the REANA system, the Kubernetes configuration must be generated. In order to do so, we should do the following.
+
+.. code-block:: sh
+
+   $ python setup.py install
+   $ reana-resources-k8s build-manifests
```

Comment:
Please add `reana-resources-k8s build-manifests` step to the Travis CI building instructions.  Otherwise we won't catch errors in shell scripts... (such as missing `pyyaml` dependency I talk about elsewhere).